docs/_data/portland-2025-sessions.yaml

```diff
@@ -177,8 +177,8 @@
   speakers:
   - name: Holly Mickelson
     slug: holly-mickelson
-    twitter:
-    website:
+    twitter: 
+    website: 
   abstract: "<p>Eventually, every technical writer is faced with the same troubling
```

Comment:
Out of scope, perhaps, but I noticed that most of the abstracts are enclosed with the `'` character, but a couple use `"`.

Reply:
No idea what that's about, and unless it breaks something, I don't think i care :-D